eng/Version.Details.xml

```diff
@@ -256,9 +256,9 @@
       <Sha>e101c379c6e3c7011562ee3af3e3de066da0048f</Sha>
       <SourceBuild RepoName="roslyn-analyzers" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="System.CommandLine" Version="2.0.0-beta1.21601.2">
+    <Dependency Name="System.CommandLine" Version="2.0.0-beta2.21623.1">
       <Uri>https://github.com/dotnet/command-line-api</Uri>
-      <Sha>8f280ecd202ab3ce64af177a6e2aa18644a26f6f</Sha>
+      <Sha>5ca753b7e262de171234ade2936796a5f0822fcb</Sha>
     </Dependency>
     <Dependency Name="Microsoft.SourceBuild.Intermediate.command-line-api" Version="0.1.257901">
```

Comment:
This dependency needs to be updated as well.  The sha should be identical to the `System.CommandLine` dependency.  There is no mechanism to enforce this.  The appropriate way to ensure everything is updated correct is to use the darc command - `darc update-dependencies --id <buildId>`

Reply:
Thanks again @MichaelSimons, I've done that just now. The DARCs docs were super helpful to get started👍